.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## Description
+[Please provide general introduction to the issue/proposal.]
+
+[If reporting a bug, attach also the entire traceback from Python. If proposing an enhancement/new feature, provide links to the related articles, reference examples, etc.]
```

Comment:
attach also -> also attach
provide links to ~~the~~ related articles, reference...